src/cargo.rs

```diff
@@ -23,27 +23,26 @@ pub enum Version {
     Latest,
     Dated(String),
 }
-pub async fn compare_versions(mock_url: Option<String>) -> Result<Version, Error> {
-    match get_latest_version(mock_url).await {
+pub async fn compare_versions(_mock_url: Option<String>) -> Result<Version, Error> {
+    match get_latest_version(_mock_url).await {
```

Comment:
Why is there an underscore before the mock_url variable @stacksjb ?

Reply:
There was a lint error due to unused variable name - I could have added the ignore dead_code parameter, but I assumed the unused variable was intended so I labeled it as such.  (It's also possible this is only used in test and this could be moved underneath test mod to resolve the error, but I didn't dig into it too deeply).

Reply:
I refactored this out, it's because I had been using the #[cfg(test)] macro which was an antipattern here.